docs/adguard-vpn-for-ios/overview.md

```diff
@@ -32,16 +32,38 @@ With AdGuard VPN, you can choose from over 50 locations in dozens of countries.
 
 ## Exclusions
 
-You can find the Exclusions tab by tapping the second icon in the bottom bar. There you will see two exclusion lists, for *Regular* and *Selective* modes.
+You can find the Exclusions tab by tapping the second icon in the bottom bar.
 
-In *Regular* mode, the VPN works for all websites except the excluded ones. Conversely, in *Selective* mode, the VPN only works for websites from the list.
+By default, AdGuard VPN is active for all websites except those added to exclusions. You can also turn it on selectively — VPN will be active only for specified websites. To switch to this mode, tap *Change mode*.
 
-You can add domains (e.g., `google.com`) or subdomains (e.g., `*.google.com`) of websites to the lists in two ways:
+> Note that each mode has its own exclusion list.
+
+You can add domains (e.g., `google.com`) or subdomains (e.g., `news.google.com`) of websites to the lists in three ways:
 
 - Enter them manually in the app
-- Add them directly from the browser by sharing the desired pages with AdGuard VPN
+- [Choose from domain lists for popular services](#domain-lists-for-popular-services)
+- Add them directly from the browser by sharing the desired pages to AdGuard VPN app
+
+![Exclusions *mobile_border](https://cdn.adtidy.org/blog/new/lpjs7iexclusions-as-of-v2-4.png)
+
+When you manually add a domain, all subdomains are added automatically (it looks like `*.google.com`). Want to keep only google.com or turn VPN off for a specific domain? Tap the line with the website — it will open a separate screen where you can manage subdomains. Just uncheck the boxes for those you don't need.
```

Comment:
```suggestion
When you manually add a domain, all subdomains are added automatically (it looks like `*.google.com`). Want to keep only google.com or turn VPN off for a specific domain? Tap the line with the website — this will open a separate screen where you can manage subdomains. Just uncheck the boxes for the ones you don't need.
```